pom.xml

```diff
@@ -122,9 +122,9 @@
       <version>2.6</version>
     </dependency>
     <dependency>
-      <groupId>org.folio</groupId>
```

Comment:
This is temporary for CI until merge of https://github.com/folio-org/folio-s3-client/pull/12